src/communication/azext_communication/manual/_help.py

```diff
@@ -111,3 +111,58 @@
         text: |-
                az communication wait --name "MyCommunicationResource" --resource-group "MyResourceGroup" --deleted
 """
+
+helps['communication identity'] = """
+    type: group
+    short-summary: Manage identity with communication
+"""
+
+helps['communication identity issue-access-token'] = """
+    type: command
+    short-summary: "create user identity and accesstoken"
+    examples:
+      - name: issue-access-token
+        text: |-
+               az communication identity issue-access-token --scope chat
+      - name: issue-access-token with multiple scopes and userid
+        text: |-
+               az communication identity issue-access-token --scope chat voip --userid "8:acs:xxxxxx"
+"""
+
+helps['communication sms'] = """
+    type: group
+    short-summary: Manage sms with communication
+"""
+
+helps['communication sms send-sms'] = """
+    type: command
+    short-summary: "Sends SMS to phone number."
+    examples:
+      - name: send sms
+        text: |-
+               az communication sms send-sms --sender "+1833xxxxxxx" \
+--recipient "+1425xxxxxxx" --message "Hello there!!"
+"""
+
+helps['communication phonenumbers'] = """
+    type: group
+    short-summary: Manage phonenumbers with communication
+"""
+
+helps['communication phonenumbers list-phonenumbers'] = """
+    type: command
+    short-summary: "List communication phone numbers"
```

Comment:
More descriptive:
Lists all phone numbers associated with the CommunicationService resource.